evaporate.js

```diff
@@ -223,8 +224,22 @@
 
         //con.simulateStalling =  true
 
-        _.add = function (file,  pConfig) {
-            var c = extend(pConfig, {});
+        _.add = function (pFile,  pConfig) {
+            var file,
+                config;
+            
+           if (arguments.length === 1) {
```

Comment:
@bikeath1337 You're a lot more familiar with this library than I am so take this with a grain of salt :) ...but would an acceptable option be to just fix the documentation to match the api? It's potentially more confusing in the long term to have two ways to pass options here.

Unless of course this fix helps maintain compatibility with older versions in which case I'd say yeah this is probably the best option.